amundsen_application/static/js/config/config-default.ts

```diff
@@ -65,40 +65,6 @@ const configDefault: AppConfig = {
           iconClass: 'icon-redshift',
         },
       },
-      filterCategories: [
-        {
-          value: 'database',
-          displayName: 'Source',
-          type: FilterType.MULTI_SELECT_VALUE,
-          options: [
-            { value: 'bigquery', displayName: 'BigQuery' },
-            { value: 'hive', displayName: 'Hive' },
-            { value: 'postgres', displayName: 'Postgres' },
-            { value: 'presto', displayName: 'Presto' },
-            { value: 'redshift', displayName: 'Redshift' },
-          ],
-        },
-        {
-          value: 'column',
-          displayName: 'Column',
-          type: FilterType.SINGLE_VALUE,
-        },
-        {
-          value: 'schema',
-          displayName: 'Schema',
-          type: FilterType.SINGLE_VALUE,
-        },
-        {
-          value: 'table',
-          displayName: 'Table',
-          type: FilterType.SINGLE_VALUE,
-        },
-        {
-          value: 'tag',
-          displayName: 'Tag',
-          type: FilterType.SINGLE_VALUE,
-        },
```

Comment:
I think we should keep the column, schema, table, and tag filters as default since those are base properties of all Amundsen tables. The database options change for each company, but maybe adding a SINGL_VALUE filter for database would be a good default.

Reply:
Sure thing I'll add these back in as the default